taglib/mpc/mpcproperties.cpp

```diff
@@ -184,7 +185,7 @@ namespace
 
   // This array looks weird, but the same as original MusePack code found at:
   // https://www.musepack.net/index.php?pg=src
-  const unsigned short sftable [8] = { 44100, 48000, 37800, 32000, 0, 0, 0, 0 };
+  constexpr std::array<unsigned short, 8> sftable { 44100, 48000, 37800, 32000, 0, 0, 0, 0 };
```

Comment:
I think this can be shortened to `constexpr std::array`. `d->sampleRate` is an `int` so requiring `sftable` to be `unsigned short` doesn't seem necessary.

Reply:
Done.